### DISCUSSION TITLE
Not supported

### DISCUSSION THREAD

**Discussion by SamHobbsOrg** [CloudBytes-Academy/CloudBytes.dev#108] [Q&A]
I was able to install a second Ubuntu using these instructions but then I got messages about it being unsupported. It said to do the upgrades for this version then upgrade to a later version. The upgrade to a later version crashed. I will download a different image and hope I can use that.

I apologize if I made a mistake.

**Comment by rehanhaider** [comment]
Hi Sam, Please can you be a bit more specific. 

I'm assuming this is about installing a second instance of Ubuntu on WSL2? 

Do you have screenshots / text of the error messages you got?
